packages/react/src/Overlay/Overlay.docs.json

```
@@ -112,6 +112,12 @@
       "defaultValue": "",
       "description": "If defined, Overlays will be rendered in the named portal. See also `Portal`."
     },
+    {
+      "name": "focusTrap",
+      "type": "string",
```

should this be a boolean instead?
```suggestion
      "type": "boolean",
```